share/functions/__fish_shared_key_bindings.fish

```diff
@@ -0,0 +1,75 @@
+function __fish_shared_key_bindings -d "Bindings shared between emacs and vi mode"
+    # These are some bindings that are supposed to be shared between vi mode and default/emacs mode
```

Comment:
We should probably start chipping away at the idea that the "default" binding mode is somehow distinct from supporting emacs bindings. There are probably a handful of places in the documentation where "default == emacs" is meaningful. But this isn't one of them.